example/src/reducers/ComponentReducer.ts

```diff
@@ -0,0 +1,36 @@
+import * as types from '../constants/ComponentActionTypes';
+import InitialState from './InitialState';
+import { IDocument, IAction } from '../Types';
+
+export default function componentReducer(state: IDocument[] = InitialState.components, action: IAction) {
+  switch (action.type) {
+    case types.REQUEST_COMPONENTS: {
+      return {
+        ...state,
+        // error: false,
```

Comment:
@mkulinski Yes. Now, i think we don't need it anymore. Later we can add it when we actually require it. Thanks for asking.